project_closing/models/analytic.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Camptocamp Author Vincent Renaville
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
```

Comment:
I see, you are right, the rule is there.
Personally, when I import several modules in the same line, the order has some meaning or logic - usually the order they should later be used. For me this usually translates to `models, fields, api`.
I'm for coding standards, but there is a point past which rules do more harm than good.